src/dvclive/live.py

```diff
@@ -663,9 +676,7 @@ def log_sklearn_plot(
         else:
             raise InvalidPlotTypeError(name)
 
-        sklearn_kwargs = {
-            k: v for k, v in kwargs.items() if k not in plot_config or k != "normalized"
-        }
+        sklearn_kwargs = {k: v for k, v in kwargs.items() if k not in plot_config}
         plot.step = self.step
         plot.dump(val, **sklearn_kwargs)
```

Comment:
```suggestion
        plot.step = self.step
        plot.dump(val, **kwargs)
```

Since the plot config options are now named args, I don't think it should be necessary to do anything here but pass `kwargs` directly.